tests/network-tests/src/fixtures/membership/InviteMembersHappyCaseFixture.ts

```diff
@@ -75,7 +75,6 @@ export class InviteMembersHappyCaseFixture extends StandardizedFixture {
       assert.equal(about, metadata.about)
       assert.equal(inviteCount, 0)
       assert.equal(avatar?.avatarUri, metadata.avatarUri || undefined)
-      assert.equal(validatorAccount, metadata.validatorAccount)
```

Comment:
Please check `assert.equal(metadata.isVerifiedValidator, false)` here too

Reply:
I can't add command. because isVerifiedValidator is not defined.